.travis/build.sh

```diff
@@ -55,11 +55,11 @@ if [ ${JAVA_MAJOR_VERSION} -eq 1 ] ; then
   EXIT=$?
   exitIfError
 
-  clearDockerEnv
-  docker pull strimzi/kafka:latest-kafka-2.4.1
-  mvn -e -V -B test -f testsuite -Pkafka-2_4_1
-  EXIT=$?
-  exitIfError
+#  clearDockerEnv
+#  docker pull strimzi/kafka:latest-kafka-2.4.1
+#  mvn -e -V -B test -f testsuite -Pkafka-2_4_1
+#  EXIT=$?
+#  exitIfError
```

Comment:
Should this be deleted? If not maybe you could add some explanatory comment.

Reply:
I added this because now the tests run much faster which gives us room to reenable testing compatibility with older versions of Kafka for anyone who might be using this without Strimzi operator. But I can remove this from Travis CI and only run these tests locally. Is the reasoning that since we don't support these versions in the operator we also don't want to support them here? For me, I simply want to know when I do some change that breaks backwards compatibility so I can consider it, and if nothing else, make a note of it in RELEASE_NOTES.md.

Reply:
The reasoning is _why do we have a code in there which is commented out_. 

So ... if this is something you want to keep to uncomment and use when needed its fine. Just add above it a comment line explaining why is there and what it is useful for. This way without any explanation is not clear what it is and why it is there and why is it commented out.

Reply:
Removed it.